### ISSUE TITLE
Keyboard-only usability: Cannot get panels to focus with keyboard

### ISSUE THREAD

**Issue by jessover9000** [grafana/grafana#20746]
**Steps to reproduce**:
If I tab through a Grafana dashboard, I can only get to left-hand navigation, toolbar or footer links. I cannot actually tab through the panels using only the keyboard or get a panel to focus.

**Actual Result**:
I cannot focus a panel in a dashboard as long as I only use the keyboard. Or if it's possible, I don't know how to do it. After I'm done tabbing through the toolbar, the focus immediately jumps to the page footer.

**Expected Result**
after finishing tabbing through the toolbar, the first panel should be in focus so that I can actually use the other shortcuts.

**Environment**:
- Grafana version: 6.5
- Data source type & version: gdev test data
- User OS & Browser: OSX, Chrome

**Comment by ckbedwell** [comment]
This issue has been resolved.

**Comment by mdbridge** [comment]
how is this resolved?  As best I can tell this problem is still present.  I am unable to focus a panel so I can use the 'e' keyboard shortcut using the keyboard.

**Comment by ckbedwell** [comment]
Sorry, I usually add the Grafana version and expand on these a bit when I am adding these comments of when they are resolved. You are right and there is a caveat to this issue being resolved in regards to the expected result being able to use the keyboard shortcuts when the panel is focussed.

The main focus of this issue is the lack of keyboard focus on panels, which you can do as of (at least) Grafana 10.2.0. The keyboard shortcut issue is separate and has an issue for it: https://github.com/grafana/grafana/issues/47005 so will track that particular problem there.

**Comment by mdbridge** [comment]
ah.  I was on 10.0.2;  I upgraded to 10.3.2 and I can now use the keyboard to focus a panel and do the shortcut from there.  Thanks!

**Comment by mdbridge** [comment]
opps.  got fooled by mouse over focus panel again -- the keyboard shortcut does not work on the focused panel.  :-(
